cmd/bootstrap/quick-start-cluster/cmd.go

```diff
@@ -0,0 +1,217 @@
+package quickstartcluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	cfTypes "github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/openshift/rosa/pkg/output"
+    "github.com/sirupsen/logrus"
+)
+
+var Cmd = makeCmd()
+
+func makeCmd() *cobra.Command {
```

Comment:
@den-rgb Rather than a command-per-quickstart, I was thinking we could use a folder structure as the quick start name and then have a generic command that works with a known folder structure. 

So for example, to support the `rosa create quickstart hcp-vpc-networking`, we'd have a folder structure like:

- quickstart/hcp-vpc-networking
- quickstart/hcp-vpc-networking/cloudformation-template.json
- quickstart/hcp-vpc-networking/metadata.json <-- Not sure if this is needed, but maybe. It could be used to keep larger descriptions of the quick start that we can't store in the CloudFormation template

So given the above structure, when the user runs `rosa create quickstart hcp-vpc-networking`, the generic command would:

- Verify that the folder exists
- Verify that the CloudFormation template exists

For parameters, the command could parse the CloudFormation template and look at the list of known/supported parameters and we allow a generic way of passing these e.g 

`rosa create quickstart hcp-vpc-network -pname=foo -ptags=foo=bar,bob=baz ...`

Very much aligned with the `oc process -f $template -p ...`

With that model, creating new quick starts is simply a case of providing new cloudformation templates (and the metadata.json) if we think we need it.